From b6e2bc1489be1d617c85529863436c8057fdde7e Mon Sep 17 00:00:00 2001
From: Ohad Ben-Cohen <ohad@wizery.com>
Date: Wed, 14 Dec 2011 13:03:55 +0200
Subject: [PATCH] rpmsg: add Kconfig menu

Add a dedicated Kconfig menu for the rpmsg drivers, so they
don't show up in the main driver menu.

Signed-off-by: Ohad Ben-Cohen <ohad@wizery.com>
---
 drivers/rpmsg/Kconfig |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/drivers/rpmsg/Kconfig b/drivers/rpmsg/Kconfig
index 811fede..98aca55 100644
--- a/drivers/rpmsg/Kconfig
+++ b/drivers/rpmsg/Kconfig
@@ -1,5 +1,9 @@
+menu "Rpmsg drivers"
+
 # RPMSG always gets selected by whoever wants it
 config RPMSG
 	tristate
 	select VIRTIO
 	select VIRTIO_RING
+
+endmenu
-- 
1.7.0.4

